lib/everypolitician/popolo/event.rb

```diff
@@ -3,6 +3,20 @@ module Popolo
     class Events < Collection; end
     class Event < Entity
       attr_accessor :start_date, :end_date
+
+      def people
+        raise NotImplementedError unless classification == 'legislative period'
```

Comment:
I don't think `NotImplementedError` is what we want to be using here. The docs seems to suggest that this is more of a Ruby internals class

```
$ ri NotImplementedError
= NotImplementedError < ScriptError

(from ruby core)
------------------------------------------------------------------------------
Raised when a feature is not implemented on the current platform. For example,
methods depending on the fsync or fork system calls may raise this exception
if the underlying operating system or Ruby runtime does not support them.

Note that if fork raises a NotImplementedError, then respond_to?(:fork)
returns false.
------------------------------------------------------------------------------

```

I think it's probably best to create our own exception that inherits from `StandardError` that we can raise instead of `NotImplementedError`.

```ruby
MethodNotImplemented = Class.new(StandardError)
```